Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt

```diff
@@ -296,7 +298,7 @@ fun SnackImage(
     ) {
         AsyncImage(
```

Comment:
Would you still need an asyncImage at all or would you consider using a typical `Image` composable instead?

Reply:
AsyncImage loads the image off the main thread, which is better for performance.